Submitted By: Ryan Oliver <ryan.oliver AT pha.com.au>
Date: 2004-10-26
Initial Package Version: 0.9.7d
Origin: 
Description: This patch fixes breakage when attempting to build against
  MIT kerberos 5 
diff -uNr openssl-0.9.7d-orig/ssl/kssl.h openssl-0.9.7d/ssl/kssl.h
--- openssl-0.9.7d-orig/ssl/kssl.h	2003-09-28 03:55:18.000000000 +1000
+++ openssl-0.9.7d/ssl/kssl.h	2004-10-26 14:44:41.000000000 +1000
@@ -81,9 +81,19 @@
 */
 #ifdef KRB5_HEIMDAL
 typedef unsigned char krb5_octet;
+#endif
+
+#ifndef FAR
 #define FAR
 #endif
 
+/*
+**	Need errno.h w MIT Krb5
+*/
+#ifdef KRB5_MIT
+#include <errno.h>
+#endif
+
 /*	Uncomment this to debug kssl problems or
 **	to trace usage of the Kerberos session key
 **
